doc/build.md

```diff
@@ -96,12 +96,15 @@ grin client help
 ```sh
 docker build -t grin -f etc/Dockerfile .
 ```
+for floonet, use `etc/Dockerfile.floonet` instead
 
 You can bind-mount your grin cache to run inside the container.
 
 ```sh
 docker run -it -d -v $HOME/.grin:/root/.grin grin
 ```
+if you prefer to use a docker named volume, you can pass `-v dotgrin:/root/.grin` instead.
+Using a named volume copies a default grin-server.toml
```

Comment:
The grin cache does not contain only the server config. Also, is this true for named volumes that already contain cached data?

Reply:
Yeah, I know it contains other data. When docker creates a new volume .. it will copy everything in the image under /root/.grin to that volume. But it will not for a bind mounted directory. When using a named volume `-v dotgrin:/root/.grin` .. if the volume contains data, it will not be overwritten